tools/slicex/slicex.go

```diff
@@ -0,0 +1,13 @@
+package slicex
+
+import "strings"
+
+// StringsContain returns true if the given value contains at least one value from the given slice
+func StringsContain(val string, sl []string) bool {
+	for _, sv := range sl {
+		if strings.Contains(val, sv) {
```

Comment:
I think it's better to match for the exact string we are looking for instead of accepting strings that contain the string as a substring. I can see that in this use case it doesn't matter but I'd rather be safe

Reply:
But the original logic uses `strings.Contain` function to find substrings. Don't really want to change an actual business logic within this PR.

Reply:
You are right!